🐙 source-couchbase: run up-to-date pipeline [2025-04-05]

airbyteio opened the pull request:
# Update source-couchbase

This PR was autogenerated by running `airbyte-ci connectors --name=source-couchbase up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Skipped

- Update versions of libraries in poetry.: Successful

- PATCH bump source-couchbase version to 0.1.3: Successful

- Build source-couchbase docker image for platform(s) linux/amd64, linux/arm64: Successful

- Get dependency updates: Successful

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful




## Dependency updates

We use [`syft`](https://github.com/anchore/syft) to generate a SBOM for the latest connector version and the one from the PR.
It allows us to spot the dependencies that have been updated at all levels and for all types of dependencies (system, python, java etc.).
Here are the dependencies that have been updated compared to `airbyte/source-couchbase:latest`.
Keep in mind that `:latest` does not always match the connector code on the main branch. 
It is the latest released connector image when the head commit of this branch was created.

| Type | Name | State | Previous Version | New Version |
|------|------|-------|-------------|------------------|
| python | cattrs | updated | 24.1.2 | **24.1.3** |
| python | orjson | updated | 3.10.15 | **3.10.16** |
| python | pydantic | updated | 2.10.6 | **2.11.2** |
| python | pydantic_core | updated | 2.27.2 | **2.33.1** |
| python | setuptools | updated | 77.0.3 | **78.1.0** |
| python | typing_extensions | updated | 4.12.2 | **4.13.1** |
| python | tzdata | updated | 2025.1 | **2025.2** |
| python | url-normalize | updated | 1.4.3 | **2.2.0** |
| python | typing-inspection | added | not present | **0.4.0** |

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=56552) <br/>All committers have signed the CLA.

airbyteio committed:
Update versions of libraries in poetry.
PATCH bump source-couchbase version to 0.1.3
Build source-couchbase docker image for platform(s) linux/amd64, linux/arm64
Get dependency updates
Create or update pull request on Airbyte repository
Add changelog entry
